app/src/main/java/com/eveningoutpost/dexdrip/Services/DexCollectionService.java

```diff
@@ -650,6 +650,10 @@ public synchronized void onServicesDiscovered(BluetoothGatt gatt, int status) {
                 }
                 status("Enabled " + getString(R.string.blukon));
                 static_use_blukon = true; // doesn't ever get unset
+
+
+                Home.setPreferencesInt("bridge_battery", 100);//force battery to max until getting right value
+                Home.setPreferencesInt("nfc_sensor_age", 1);//in min, set to max first
```

Comment:
I think the `nfc_sensor_age`  should be set to 0 here. This code is executed after each service discovery which happens when the bluetooth disconnects and reconnects. If there is no reading after this and the user opens the app they will see the wrong sensor age. 0 implies no-value and nothing will be displayed if it is set to that.